openapi-contracts/src/main/com/sinch/sdk/domains/verification/models/v1/status/response/SmsVerificationStatusResponsePrice.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Verification | Sinch
+ * Verify users with SMS, flash calls (missed calls), a regular call, or data verification.
+ *
+ * The version of the OpenAPI document: 2.0.1
+ * Contact: support@sinch.com
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+package com.sinch.sdk.domains.verification.models.v1.status.response;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.sinch.sdk.domains.verification.models.v1.Price;
+
+/** Prices associated with this verification */
+@JsonDeserialize(builder = SmsVerificationStatusResponsePriceImpl.Builder.class)
+public interface SmsVerificationStatusResponsePrice {
```

Comment:
This class gets a long name as this structure for the SMS price is used for a text message only. On the opposite, the price for a PSTN call (phone call or flash call) has its own naming `StatusPrice`. Can we imagine to rename them in order to make the distinction more easily?
Suggestion: `StatusCallPrice` or `StatusPstnPrice` and `StatusTextPrice` or `StatusSmsPrice`

Reply:
Yes.
Will be added to the next PR: more related to DX for SDK usage in place off just "migrating" got new generator